stable/fluent-bit/templates/secret.yaml

```diff
@@ -12,3 +13,18 @@ type: Opaque
 data:
   es-tls-ca.crt: {{ .Values.backend.es.tls_ca | b64enc | quote }}
 {{- end }}
+{{- if .Values.backend.stackdriver.google_service_credentials }}
+---
```

Comment:
Shouldn't you check if the backend type is stackdriver?

Reply:
Got it. Fixed.